src/bin/compute-wer.cc

```diff
@@ -69,29 +34,26 @@ int main(int argc, char *argv[]) {
         "Compute WER by comparing different transcriptions\n"
         "Takes two transcription files, in integer or text format,\n"
         "and outputs overall WER statistics to standard output.\n"
-        "Optionally, the third argument can be used to obtain detailed statistics\n"
         "\n"
-        "Usage: compute-wer [options] <ref-rspecifier> <hyp-rspecifier> [<stats-out>]\n"
+        "Usage: compute-wer [options] <ref-rspecifier> <hyp-rspecifier>\n"
         "\n"
-        "E.g.: compute-wer --text --mode=present ark:data/train/text ark:hyp_text\n"
-        "or: compute-wer --text --mode=present ark:data/train/text ark:hyp_text - | \\\n"
-        "   sort | uniq -c\n";
+        "E.g.: compute-wer --text --mode=present ark:data/train/text ark:hyp_text\n";
```

Comment:
Please add:
  "See also: align-text\n"
to the usage message.
It would also be useful if 'align-text', in its usage message, referred to the relevant scripts.